sousmot/sousmotapp/static/scss/main.scss

```
@@ -65,25 +65,68 @@ $navbar-item-active-color: $navbar-item-hover-color;
     }
 
     @include mobile {
-      max-width: 80vw;
+        max-width: 80vw;
     }
 }
 
 footer.footer {
     text-align: center;
-    padding:3rem 1.5rem 4rem;
+    padding: 3rem 1.5rem 4rem;
 }
 
 body {
-    display:flex;
+    display: flex;
     min-height: 100vh;
     flex-direction: column;
 }
 
 section {
-    flex:1;
+    flex: 1;
 }
 
 .has-addons-fullwidth .button.is-fullwidth {
-    width:50%;
-}
\ No newline at end of file
+    width: 50%;
+}
+
+// Game
+.table td {
+    padding: 0.5rem;
+    color: white;
```

```suggestion
    color: $text;
```
Maybe use the same color has the text ?